backend/src/api/groups/groups.api.ts

```diff
@@ -61,6 +63,19 @@ const initGroupsApi: FastifyPluginAsync<Options> = async (fastify, opts) => {
       return rep.status(HttpCode.OK).send(groups);
     },
   });
+
+  fastify.route({
+    method: HttpMethod.PUT,
+    url: GroupsApiPath.$ID,
+    schema: {
+      body: groupUpdateValidationSchema,
+    },
+    async handler(req: FastifyRequest<{ Body: GroupsUpdateRequestDto }>, rep) {
+      const group = await groupService.update(req.body);
```

Comment:
As you declared `GroupsApiPath.$ID` route contains `:id` - so you should take group id from params (also provide validation for params) and wait from request body only payload for group update